routes/api-routes.js

```diff
@@ -0,0 +1,53 @@
+const express = require("express");
```

Comment:
good job importing all of your external modules and packages at the top of the file! This is great for readability :-)